stock_request/models/stock_request_order.py

```diff
@@ -236,9 +236,12 @@ def action_draft(self):
 
     def action_cancel(self):
         self.mapped("stock_request_ids").action_cancel()
-        self.write({"state": "cancel"})
+        self._action_cancel()
         return True
 
+    def _action_cancel(self):
```

Comment:
The same. It can be done in the same method and you can overwrite such method.

Reply:
I mean in `action_cancel` inside `stock.request.order`. Don't do `action_cancel` and `_action_cancel`.

Reply:
Ok, `_action_cancel()` method is already removed.